pkg/tests/ossm/smoke_test.go

```diff
@@ -117,6 +115,14 @@ func TestSmoke(t *testing.T) {
 			})
 		})
 
+		t.NewSubTest(fmt.Sprintf("install smcp %s", toVersion)).Run(func(t TestHelper) {
+			t.Logf("This test checks whether SMCP %s deletion deletes all the resources", env.GetSMCPVersion())
+
+			t.LogStepf("Install SMCP %s", toVersion)
+			assertSMCPDeploysAndIsReady(t, toVersion)
+			assertRoutesExist(t)
```

Comment:
This subtest should also install bookinfo and test if traffic flows through it, and maybe also test proxy start up time.  Basically, the assertions should be exactly the same as in the upgrade subtest.

Reply:
Ok, done